### PR TITLE
Some tweaks to `ImageEditor`

### PR DESCRIPTION
**Pull request by abidlabs**
Some small tweaks to documentation, preprocess, and templates

### CODE REVIEW

**Comment by aliabd** [comment]
@abidlabs I'm a bit confused why you added the imageeditor stuff to the 4.0 changelog?

**Review by hysts — APPROVED** [review]
LGTM!

File: gradio/components/image_editor.py [hunk 1]
```diff
@@ -234,7 +235,7 @@ def preprocess(self, x: EditorData | None) -> EditorValue | None:
         if x is None:
             return x
 
-        bg = self.convert_and_format_image(x.background, crop_size=self.crop_size)
+        bg = self.convert_and_format_image(x.background)
```
Comment (pngwn) [inline gradio/components/image_editor.py:238]: Ah nice catch. Didn't realise i screwed this up, explains the weird behaviour.

**Comment by aliabd** [comment]
hmm @abidlabs not sure that makes too much sense, maybe we should create a separate migration to gradio 4 document, but idk if it makes sense adding it as a changelog entry to a version that doesn't support it. Maybe you should add a qualifier ("supported in 4.5") or something? Probably not a big deal

**Comment by abidlabs** [comment]
@aliabd I do mention "as of Gradio 4.5.0" -- do you have something else in mind?

**Review by aliabd — APPROVED** [review]

**Review by freddyaboulton — APPROVED** [review]
I think this should be a patch no? Otherwise, looks good to me! Thanks @abidlabs !

**Comment by abidlabs** [comment]
Thanks everyone for the reviews! I'll go ahead and merge it in after CI runs

**Pull request merged**